docs/conf.py

```diff
@@ -38,7 +38,7 @@
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #
-needs_sphinx = '1.6.3'
+needs_sphinx = '3.0.0'
```

Comment:
Aha, and I was always wondering why rtd was not using  a more recent version of Sphinx :-)